src/pybind/mgr/cli_api/tests/test_cli_api.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from ..module import CLI, BenchmarkException, HandleCommandResult
+
+
+class BenchmarkRunnerTest(unittest.TestCase):
+    def test_number_of_calls_on_start_fails(self):
+        with self.assertRaises(BenchmarkException) as ctx:
+            CLI('CLI', 0, 0).benchmark(0, 10, 'list_servers', [])
```

Comment:
since you are creating the same cli always you can extract this.
```suggestion
    def setUp(self):
        self.cli = CLI('CLI', 0, 0)
    def test_number_of_calls_on_start_fails(self):
        with self.assertRaises(BenchmarkException) as ctx:
            self.cli.benchmark(0, 10, 'list_servers', [])
```